lib/api/documentExtractor.js

```diff
@@ -39,7 +39,7 @@ const extractors = ([
         return request;
       }
     },
-    targets: ['delete', 'get']
+    targets: ['delete', 'get', 'exists']
```

Comment:
Well `exists` did not technically returns a document but it still get info about this document so we can add it, good job :+1: